src/main/java/kekztech/common/tileentities/MTELapotronicSuperCapacitor.java

```diff
@@ -1522,31 +1522,20 @@ public PlaceResult survivalPlaceBlock(MTELapotronicSuperCapacitor t, World world
             ItemStack trigger, AutoPlaceEnvironment env) {
             if (check(t, world, x, y, z)) return PlaceResult.SKIP;
             int glassTier = GTStructureChannels.BOROGLASS.getValue(trigger) + 2;
+            // 0 is None in the Capacitors; So we start at 1; Glasstier -3 since the LSC allows capacitor to be at most
+            // 3 levels underneath the glass level
+            int capacitorTier = Math.max(1, Math.min(Capacitor.VALUES_BY_TIER.length - 1, glassTier - 3));
```

Comment:
Should this be the following? The glass check on line 1530 always succeeds from what I see.
```suggestion
            int capacitorTier = getHint(trigger);
```

Reply:
will have a look at it tomorrow or the day after. sorry. Didn't see your comment